### ISSUE TITLE
Can't use global $_ in "my" at /usr/share/perl5/FusqlFS/Cache/Limited.pm line 194, near "my $_ "

### ISSUE THREAD

**Issue by eMPee584** [kstep/fusqlfs#17]
```
Can't use global $_ in "my" at /usr/share/perl5/FusqlFS/Cache/Limited.pm line 194, near "my $_ "                                                              
Compilation failed in require at /usr/share/perl5/FusqlFS/Cache.pm line 50.
BEGIN failed--compilation aborted at /usr/share/perl5/FusqlFS/Cache.pm line 50.                                                                               
Compilation failed in require at /usr/share/perl5/FusqlFS.pm line 47.
BEGIN failed--compilation aborted at /usr/share/perl5/FusqlFS.pm line 47.
Compilation failed in require at /usr/bin/fusqlfs line 13.
BEGIN failed--compilation aborted at /usr/bin/fusqlfs line 13.
Press ENTER to continue
Can't use global $_ in "my" at /usr/share/perl5/FusqlFS/Cache/Limited.pm line 194, near "my $_ "                                                              
Compilation failed in require at /usr/share/perl5/FusqlFS/Cache.pm line 50.
BEGIN failed--compilation aborted at /usr/share/perl5/FusqlFS/Cache.pm line 50.                                                                               
Compilation failed in require at /usr/share/perl5/FusqlFS.pm line 47.
BEGIN failed--compilation aborted at /usr/share/perl5/FusqlFS.pm line 47.
Compilation failed in require at /usr/bin/fusqlfs line 13.
BEGIN failed--compilation aborted at /usr/bin/fusqlfs line 13.
```
#sad

**Comment by kstep** [comment]
Wow! Somebody decided to use this ancient piece of my code! Thank you for your attention, kind man! )
I ran this code 6 code ago, I guess Perl moved on since that time.

**Comment by kstep** [comment]
Fixed by #18
